.github/actions/javascript/reviewerChecklist/reviewerChecklist.js

```diff
@@ -0,0 +1,106 @@
+const core = require('@actions/core');
+const github = require('@actions/github');
+const _ = require('underscore');
+const https = require('https');
+const GitHubUtils = require('../../../libs/GithubUtils');
+
+const pathToReviewerChecklist = 'https://raw.githubusercontent.com/Expensify/App/main/contributingGuides/REVIEWER_CHECKLIST.md';
+const reviewerChecklistStartsWith = '## Reviewer Checklist';
+const issue = github.context.payload.issue ? github.context.payload.issue.number : github.context.payload.pull_request.number;
+const combinedComments = [];
+
+/**
+ * @returns {Promise}
+ */
+function getNumberOfItemsFromReviewerChecklist() {
```

Comment:
Could DRY this up by making it take the filepath and then putting it in `.github/libs`

Reply:
Since there is custom logic for the author checklist to parse out the checklist, I don't think it makes DRYing this up very valuable.